src/content/learn/escape-hatches.md

```diff
@@ -191,47 +191,48 @@ Read **[Synchronizing with Effects](/learn/synchronizing-with-effects)** to lear
 
 </LearnMore>
 
-## You Might Not Need An Effect {/*you-might-not-need-an-effect*/}
+## Kemungkinan besar anda tidak memerlukan Efek {/*you-might-not-need-an-effect*/}
 
-Effects are an escape hatch from the React paradigm. They let you "step outside" of React and synchronize your components with some external system. If there is no external system involved (for example, if you want to update a component's state when some props or state change), you shouldn't need an Effect. Removing unnecessary Effects will make your code easier to follow, faster to run, and less error-prone.
+Fitur Efek sebetulnya tidak sejalan dengan paradigma React. Fitur ini memungkinkan Anda "keluar" dari React dan menyinkronkan komponen-komponen Anda dengan sistem eksternal tertentu. Jika tidak ada sistem eksternal yang terlibat (misalnya, jika Anda ingin memperbarui *state* komponen ketika beberapa *props* atau *state* berubah), Anda seharusnya tidak memerlukan Efek. Menghapus Efek yang tidak perlu akan membuat kode Anda lebih mudah dipahami, lebih cepat dijalankan, dan mengurangi potensi kesalahan.
 
-There are two common cases in which you don't need Effects:
-- **You don't need Effects to transform data for rendering.**
-- **You don't need Effects to handle user events.**
+Terdapat dua kasus umum dimana Anda tidak akan memerlukan Efek:
+- **Anda tidak memerlukan Efek untuk mentransformasi data yang akan dirender**
+- **Anda tidak memerlukan Efek untuk menangani *event* dari pengguna**
 
-For example, you don't need an Effect to adjust some state based on other state:
+
+Contohnya, Anda tidak memerlukan Efek untuk mengubah *state* berdasarkan *state* lain:
 
 ```js {5-9}
 function Form() {
-  const [firstName, setFirstName] = useState('Taylor');
-  const [lastName, setLastName] = useState('Swift');
+  const [namaDepan, setNamaDepan] = useState('Isyana');
```

Comment:
Have to confirm with the others @resir014 @mazipan @r17x @zainfathoni whether to localize the person name or not:

Taylor Swift vs Isyana Sarasvati

Because this might affect other upcoming PRs with related cases.

Reply:
Also I suggest not to translate the variable names

Let's keep it as `firstName, setFirstName`

@resir014 @mazipan @r17x @zainfathoni please confirm

Reply:
To simplify in keeping everything in sync, let's keep names at it is.